reports-backend/src/main/java/org/jboss/da/products/impl/AggregatedProductProvider.java

```diff
@@ -35,7 +35,7 @@
  *
  * @author Honza Brázdil &lt;jbrazdil@redhat.com&gt;
  */
-@ApplicationScoped
+//@ApplicationScoped // Should be Request scoped
```

Comment:
Stupid question: does injection work when there are no annotation on the class?

Reply:
Yup. The default scope is [Dependent](https://docs.jboss.org/cdi/api/1.0/javax/enterprise/context/Dependent.html), which works here now, but the Request scope would be better. However Request scope doesn't work with websockets.

Reply:
Alright! I haven't found any doc on that, but I guess it's not even documented(?)

Reply:
https://docs.oracle.com/javaee/6/tutorial/doc/gjbbk.html
> Dependent
> @Dependent
> The default scope if none is specified; it means that an object exists to serve exactly one client (bean) and has the same lifecycle as that client (bean).